simapp/helpers/service_msg_client.go

```diff
@@ -0,0 +1,50 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+
+	gogogrpc "github.com/gogo/protobuf/grpc"
+	grpc "google.golang.org/grpc"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ gogogrpc.ClientConn = &ServiceMsgClientConn{}
+
+// ServiceMsgClientConn is an instance of grpc.ClientConn that is used to test building
+// transactions with MsgClient's. It is intended to be replaced by the work in
+// https://github.com/cosmos/cosmos-sdk/issues/7541 when that is ready.
+type ServiceMsgClientConn struct {
```

Comment:
this is a duplicate struct from what's in x/bank/client/testutil.

Could we:
- move this to e.g. `types/msgservice`
- x/bank/client/testutil also use that one?